src/main/io/serial_cli.c

```diff
@@ -2042,6 +2065,82 @@ static void cliFeature(char *cmdline)
     }
 }
 
+#ifdef BEEPER
+static void cliBeeper(char *cmdline)
+{
+    uint32_t i;
+    uint32_t len = strlen(cmdline);;
+    uint8_t beeperCount = beeperTableEntryCount();
+    uint32_t mask = getBeeperOffMask();
+
+    if (len == 0) {
+        cliPrintf("Disabled:");
+        for (int i = 0; ; i++) {
+            if (i == beeperCount - 2) {
+                if (mask == 0)
+                    cliPrint("  none");
+                break;
+            }
+            if (mask & (1 << i))
+                cliPrintf("  %s", beeperNameForTableIndex(i));
+        }
+        cliPrint("\r\n");
+        return;
+    }
+   while (cmdline[0] == ' ') {
+        cmdline++;
+        len--;
+    }
+    if (strncasecmp(cmdline, "list", len) == 0) {
+        cliPrint("Available:");
+        for (i = 0; i < beeperCount; i++)
+            cliPrintf("  %s", beeperNameForTableIndex(i));
+        cliPrint("\r\n");
+        return;
+    } else {
+        bool remove = false;
+        if (cmdline[0] == '-') {
+            remove = true;     // this is for beeper OFF condition
+            cmdline++;
+            len--;
+        }
+        while (cmdline[0] == ' ') {
+            cmdline++;
+            len--;
+        }
+
+        for (i = 0; ; i++) {
+            if (i == beeperCount) {
+                cliPrint("Invalid name\r\n");
+                break;
+            }
+            if (strncasecmp(cmdline, beeperNameForTableIndex(i), len) == 0) {
+                if (remove) { // beeper off
+                    if (i == BEEPER_ALL - 1) {
+                        beeperOffSetAll(beeperCount - 2);
```

Comment:
I want this beeper command badly so I tried this PR and I found this got to be beeperCount - 1 or otherwise "ALL" won't set/unset SYSTEM_INIT. Also ledvinap's fix above for another instance of beeperCount - 2 is needed in order to show SYSTEM_INIT correctly.